static/js/components/dashboard/CourseAction.js

```diff
@@ -103,12 +103,12 @@ export default class CourseAction extends React.Component {
   }
 
   renderDescription = R.curry(
-    (className: string, runStatus: ?string, text: string): React$Element<*>|null => {
+    (className: string, runStatus: ?string, text: ?string): React$Element<*>|null => {
```

Comment:
It looks like Flow doesn't know how to properly check the types of functions that are wrapped with `R.curry`, I was able to change `className: string` to `className: number` without any errors. So, I think it's good to document types for readability, but keep in mind that Flow won't catch errors here :(

Maybe they're working on stuff like that! I hope so.